code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -292,7 +292,7 @@
 		if(!ishuman(target) || iscultist(target))
 			return
 		var/mob/living/carbon/human/H = target
-		H.hallucination = max(H.hallucination, 120)
+		H.Hallucinate(120 SECONDS)
```

Comment:
Is this wise? If they are already hallucinating for over 120 seconds for some reason, this will reduce the time they are hallucinating.